packages/runner-sdk/lib/sync.ts

```diff
@@ -17,6 +21,17 @@ export abstract class NangoSyncBase extends NangoActionBase {
         if (config.track_deletes) {
             this.track_deletes = config.track_deletes;
         }
+
+        if (config.syncVariant) {
+            this.variant = config.syncVariant;
+        }
+    }
+
+    public modelFullName(model: string) {
+        if (this.variant === BASE_VARIANT) {
+            return model;
+        }
+        return `${model}::${this.variant}`;
     }
```

Comment:
Does it need to be inside the class? we can put helpers outside now

Reply:
it needs to access `this.variant`